### DISCUSSION TITLE
Volume Slider Steps Setting

### DISCUSSION THREAD

**Discussion by ricostap** [nwg-piotr/nwg-shell#417] [Q&A] [answered]
First of all a big shout out to Pjotr - Thank you for developing nwg-shell! An incredible piece of software and the (for me) best addition to my arch based hyprland set up. My computer has never been as functional or as enjoyable as it is today.

My question: When I use the scroll wheel to adjust output sound volume, the adjustment happens in steps of 15. I would like to adjust that number to a lesser one. I looked at some config files and did a web search but couldn't find any information as to where to modify this setting. Can someone kindly point me in the right direction?

**Comment by nwg-piotr** [comment]
Hi Rico,

Thank you for you kind words! :)

If it comes to the issue: you're right, it annoys me too. ~~I'll add a field to adjust the [page increment](https://lazka.github.io/pgi-docs/Gtk-3.0/classes/Range.html#Gtk.Range.set_increments) value, common for all sliders in the Controls module.~~

The above does not work as expected, as the method arguments are floats, with no respect to actual changes on the slider. I'll just set step and page increment values to 0.5, that seems to result in ability to control sliders at the step of 1%.

**Reply by nwg-piotr** [reply, marked as answer]
Done: https://github.com/nwg-piotr/nwg-panel/pull/332
